modules/digiTrustIdSystem.md

```diff
@@ -0,0 +1,74 @@
+## DigiTrust Universal Id Integration
+
+Setup
+-----
+The DigiTrust Id integration for Prebid may be used with or without the full
+DigiTrust library. This is an optional module that must be used in conjunction
+with the userId module.
+
+The full DigiTrust integration requires the DigiTrust package be included.
+DigiTrust may be initialized in the standard manner or through the params
+object. 
+
+You can use npm or reference the script on the cdn.
+
+npm install digitrust
+
+or
+
+<script src="https://cdn.digitru.st/prod/1.5.20/digitrust.min.js"></script>
+
+Please note that version 1.5.20 or better is required.
+
+
+See the [Prebid Integration Guide for DigiTrust](https://github.com/digi-trust/dt-cdn/wiki/Prebid-Integration-for-DigiTrust-Id)
+and the [DigiTrust wiki](https://github.com/digi-trust/dt-cdn/wiki)
+for further instructions.
+
+
+## Example Prebid Configuration for Digitrust Id
+```
+        pbjs.que.push(function() {
+            pbjs.setConfig({
+                usersync: {
+                    userIds: [{
+						name: "digitrust",
+						params: {
+							init: {
+								member: 'example_member_id',
+								site: 'example_site_id'
+							},
+							callback: function (digiTrustResult) {
+								if (digiTrustResult.success) {
```

Comment:
suggest removing the callback logging. people copy and paste this into their page and then ask me "why is it logging all this stuff". examples should probably not include console.log at all.

Reply:
OK